### ISSUE TITLE
After updating my mac to 14.5 my docker desktop is no longer able to run certain images

### ISSUE THREAD

**Issue by BradyBlackley** [docker/for-mac#7318]
### Description

After updating my MacBook Pro to Sonoma 14.5 my docker desktop is no longer able to run certain images. After some research it appears the latest upgrade of Mac OS may break Rosetta. I currently can't run my containerized unit tests in a Spring Boot application that use Docker Desktop. I am on v4.30.0 of Docker Desktop. 

Some references:
https://www.reddit.com/r/docker/comments/1czh2wz/failed_to_start_emulated_container_on_sonoma_145/
https://github.com/docker/for-mac/issues/7006#issuecomment-1766938364

### Reproduce

Hit play button on [rmohr/activemq](https://hub.docker.com/r/rmohr/activemq) image or [confluentinc/cp-kafka](https://hub.docker.com/r/confluentinc/cp-kafka) within docker desktop after pulling them and you will see the containers end with an exit status

### Expected behavior

You should see something like this: "A fatal error has been detected by the Java Runtime Environment" in the logs

```
2024-06-06 13:47:56 INFO: Loading '/opt/activemq/bin/env'
2024-06-06 13:47:56 INFO: Using java '/usr/local/openjdk-8/bin/java'
2024-06-06 13:47:56 INFO: Starting in foreground, this is just for debugging purposes (stop process by pressing CTRL+C)
2024-06-06 13:47:56 INFO: Creating pidfile /opt/activemq/data/activemq.pid
2024-06-06 13:47:56 Java Runtime: Oracle Corporation 1.8.0_212 /usr/local/openjdk-8
2024-06-06 13:47:56   Heap sizes: current=62976k  free=58706k  max=932352k
2024-06-06 13:47:56     JVM args: -Xms64M -Xmx1G -Djava.util.logging.config.file=logging.properties -Djava.security.auth.login.config=/opt/activemq/conf/login.config -Dcom.sun.management.jmxremote -Djava.awt.headless=true -Djava.io.tmpdir=/opt/activemq/tmp -Dactivemq.classpath=/opt/activemq/conf:/opt/activemq/../lib/: -Dactivemq.home=/opt/activemq -Dactivemq.base=/opt/activemq -Dactivemq.conf=/opt/activemq/conf -Dactivemq.data=/opt/activemq/data
2024-06-06 13:47:56 Extensions classpath:
2024-06-06 13:47:56   [/opt/activemq/lib,/opt/activemq/lib/camel,/opt/activemq/lib/optional,/opt/activemq/lib/web,/opt/activemq/lib/extra]
2024-06-06 13:47:56 ACTIVEMQ_HOME: /opt/activemq
2024-06-06 13:47:56 ACTIVEMQ_BASE: /opt/activemq
2024-06-06 13:47:56 ACTIVEMQ_CONF: /opt/activemq/conf
2024-06-06 13:47:56 ACTIVEMQ_DATA: /opt/activemq/data
2024-06-06 13:47:56 Loading message broker from: xbean:activemq.xml
2024-06-06 13:47:57  INFO | Refreshing org.apache.activemq.xbean.XBeanBrokerFactory$1@1a968a59: startup date [Thu Jun 06 19:47:57 UTC 2024]; root of context hierarchy
2024-06-06 13:47:57 #
2024-06-06 13:47:57 # A fatal error has been detected by the Java Runtime Environment:
2024-06-06 13:47:57 #
2024-06-06 13:47:57 #  SIGSEGV (0xb) at pc=0x00007ffffe5ad613, pid=18, tid=0x00007fffe89f9700
2024-06-06 13:47:57 #
2024-06-06 13:47:57 # JRE version: OpenJDK Runtime Environment (8.0_212-b04) (build 1.8.0_212-b04)
2024-06-06 13:47:57 # Java VM: OpenJDK 64-Bit Server VM (25.212-b04 mixed mode linux-amd64 compressed oops)
2024-06-06 13:47:57 # Problematic frame:
2024-06-06 13:47:57 # V  [libjvm.so+0x917613]  oopDesc* PSPromotionManager::copy_to_survivor_space<false>(oopDesc*)+0xf3
2024-06-06 13:47:57 #
2024-06-06 13:47:57 # Failed to write core dump. Core dumps have been disabled. To enable core dumping, try "ulimit -c unlimited" before starting Java again
2024-06-06 13:47:57 #
2024-06-06 13:47:57 # An error report file with more information is saved as:
2024-06-06 13:47:57 # /opt/apache-activemq-5.15.9/hs_err_pid18.log
2024-06-06 13:47:57 #
2024-06-06 13:47:57 # If you would like to submit a bug report, please visit:
2024-06-06 13:47:57 #   http://bugreport.java.com/bugreport/crash.jsp
2024-06-06 13:47:57 #
2024-06-06 13:47:57 Aborted
```

### docker version

```bash
Client:
 Cloud integration: v1.0.35+desktop.13
 Version:           26.1.1
 API version:       1.45
 Go version:        go1.21.9
 Git commit:        4cf5afa
 Built:             Tue Apr 30 11:44:56 2024
 OS/Arch:           darwin/arm64
 Context:           desktop-linux

Server: Docker Desktop 4.30.0 (149282)
 Engine:
  Version:          26.1.1
  API version:      1.45 (minimum version 1.24)
  Go version:       go1.21.9
  Git commit:       ac2de55
  Built:            Tue Apr 30 11:48:04 2024
  OS/Arch:          linux/arm64
  Experimental:     false
 containerd:
  Version:          1.6.31
  GitCommit:        e377cd56a71523140ca6ae87e30244719194a521
 runc:
  Version:          1.1.12
  GitCommit:        v1.1.12-0-g51d5e94
 docker-init:
  Version:          0.19.0
  GitCommit:        de40ad0
```

### docker info

```bash
Client:
 Version:    26.1.1
 Context:    desktop-linux
 Debug Mode: false
 Plugins:
  buildx: Docker Buildx (Docker Inc.)
    Version:  v0.14.0-desktop.1
    Path:     /Users/shf7933/.docker/cli-plugins/docker-buildx
  compose: Docker Compose (Docker Inc.)
    Version:  v2.27.0-desktop.2
    Path:     /Users/shf7933/.docker/cli-plugins/docker-compose
  debug: Get a shell into any image or container (Docker Inc.)
    Version:  0.0.29
    Path:     /Users/shf7933/.docker/cli-plugins/docker-debug
  dev: Docker Dev Environments (Docker Inc.)
    Version:  v0.1.2
    Path:     /Users/shf7933/.docker/cli-plugins/docker-dev
  extension: Manages Docker extensions (Docker Inc.)
    Version:  v0.2.23
    Path:     /Users/shf7933/.docker/cli-plugins/docker-extension
  feedback: Provide feedback, right in your terminal! (Docker Inc.)
    Version:  v1.0.4
    Path:     /Users/shf7933/.docker/cli-plugins/docker-feedback
  init: Creates Docker-related starter files for your project (Docker Inc.)
    Version:  v1.1.0
    Path:     /Users/shf7933/.docker/cli-plugins/docker-init
  sbom: View the packaged-based Software Bill Of Materials (SBOM) for an image (Anchore Inc.)
    Version:  0.6.0
    Path:     /Users/shf7933/.docker/cli-plugins/docker-sbom
  scout: Docker Scout (Docker Inc.)
    Version:  v1.8.0
    Path:     /Users/shf7933/.docker/cli-plugins/docker-scout

Server:
 Containers: 3
  Running: 1
  Paused: 0
  Stopped: 2
 Images: 5
 Server Version: 26.1.1
 Storage Driver: overlay2
  Backing Filesystem: extfs
  Supports d_type: true
  Using metacopy: false
  Native Overlay Diff: true
  userxattr: false
 Logging Driver: json-file
 Cgroup Driver: cgroupfs
 Cgroup Version: 2
 Plugins:
  Volume: local
  Network: bridge host ipvlan macvlan null overlay
  Log: awslogs fluentd gcplogs gelf journald json-file local splunk syslog
 Swarm: inactive
 Runtimes: io.containerd.runc.v2 runc
 Default Runtime: runc
 Init Binary: docker-init
 containerd version: e377cd56a71523140ca6ae87e30244719194a521
 runc version: v1.1.12-0-g51d5e94
 init version: de40ad0
 Security Options:
  seccomp
   Profile: unconfined
  cgroupns
 Kernel Version: 6.6.26-linuxkit
 Operating System: Docker Desktop
 OSType: linux
 Architecture: aarch64
 CPUs: 12
 Total Memory: 7.657GiB
 Name: docker-desktop
 ID: 9f503aa3-3e0b-4769-868e-facc4a8689b3
 Docker Root Dir: /var/lib/docker
 Debug Mode: false
 HTTP Proxy: http.docker.internal:3128
 HTTPS Proxy: http.docker.internal:3128
 No Proxy: hubproxy.docker.internal
 Labels:
  com.docker.desktop.address=unix:///Users/shf7933/Library/Containers/com.docker.docker/Data/docker-cli.sock
 Experimental: false
 Insecure Registries:
  hubproxy.docker.internal:5555
  127.0.0.0/8
 Live Restore Enabled: false

WARNING: daemon is not using the default seccomp profile
```

### Diagnostics ID

EA80ED6B-9259-404E-9652-0DC0EDC9E52C/20240606195430

### Additional Info

```
         ▄                                                                                        
     ▄ ▄ ▄  ▀▄▀                                                                                   
   ▄ ▄ ▄ ▄ ▄▇▀  █▀▄ █▀█ █▀▀ █▄▀ █▀▀ █▀█                                                           
  ▀████████▀    █▄▀ █▄█ █▄▄ █ █ ██▄ █▀▄                                                           
   ▀█████▀                        DEBUG                                                           
                                                                                                  
Builtin commands:                                                                                 
- install [tool1] [tool2] ...    Add Nix packages from: https://search.nixos.org/packages         
- uninstall [tool1] [tool2] ...  Uninstall NixOS package(s).                                      
- entrypoint                     Print/lint/run the entrypoint.                                   
- builtins                       Show builtin commands.                                           
                                                                                                  
Checks:                                                                                           
✓ distro:            Debian GNU/Linux 9 (stretch)                                                 
✓ entrypoint linter: no errors (run 'entrypoint' for details)                                     
                                                                                                  
Note: This is a sandbox shell. All changes will not affect the actual container.                  
                                                                            Version: 0.0.29 (BETA)
root@ad5e5b71d598 /opt/apache-activemq-5.15.9 [quizzical_wilbur]
docker > cat /opt/apache-activemq-5.15.9/hs_err_pid19.log
#
# A fatal error has been detected by the Java Runtime Environment:
#
#  Internal Error (nmethod.cpp:2659), pid=19, tid=0x00007fffcabf7700
#  guarantee(nm->_lock_count >= 0) failed: unmatched nmethod lock/unlock
#
# JRE version: OpenJDK Runtime Environment (8.0_212-b04) (build 1.8.0_212-b04)
# Java VM: OpenJDK 64-Bit Server VM (25.212-b04 mixed mode linux-amd64 compressed oops)
# Failed to write core dump. Core dumps have been disabled. To enable core dumping, try "ulimit -c unlimited" before starting Java again
#
# If you would like to submit a bug report, please visit:
#   http://bugreport.java.com/bugreport/crash.jsp
#

---------------  T H R E A D  ---------------

Current thread (0x00007ffff859d000):  JavaThread "ActiveMQ Transport Server: stomp://0.0.0.0:61613?maximumConnections=1000&wireFormat.maxFrameSize=104857600" daemon [_thread_in_vm, id=87, stack(0x00007fffcaaf7000,0x00007fffcabf8000)]

Stack: [0x00007fffcaaf7000,0x00007fffcabf8000],  sp=0x00007fffcabf4cb0,  free space=1015k
Native frames: (J=compiled Java code, j=interpreted, Vv=VM code, C=native code)
V  [libjvm.so+0xa47bcd]  VMError::report_and_die()+0x15d
V  [libjvm.so+0x4e0492]  report_vm_error(char const*, int, char const*, char const*)+0x62
V  [libjvm.so+0x87b06f]  nmethodLocker::unlock_nmethod(nmethod*)+0x6f
V  [libjvm.so+0x9500fd]  SharedRuntime::resolve_sub_helper(JavaThread*, bool, bool, Thread*)+0x3ed
V  [libjvm.so+0x95053f]  SharedRuntime::resolve_helper(JavaThread*, bool, bool, Thread*)+0x3f
V  [libjvm.so+0x950a8c]  SharedRuntime::resolve_opt_virtual_call_C(JavaThread*)+0x3c
v  ~RuntimeStub::resolve_opt_virtual_call
J 3254 C1 java.net.SocketTimeoutException.<init>(Ljava/lang/String;)V (6 bytes) @ 0x00007fffe92a7738 [0x00007fffe92a75e0+0x158]
v  ~StubRoutines::call_stub
V  [libjvm.so+0x662d8b]  JavaCalls::call_helper(JavaValue*, methodHandle*, JavaCallArguments*, Thread*)+0x108b
V  [libjvm.so+0x6639df]  JavaCalls::call_special(JavaValue*, KlassHandle, Symbol*, Symbol*, JavaCallArguments*, Thread*)+0x2af
V  [libjvm.so+0x55fba3]  Exceptions::new_exception(Thread*, Symbol*, Symbol*, JavaCallArguments*, Handle, Handle)+0x173
V  [libjvm.so+0x5602e3]  Exceptions::new_exception(Thread*, Symbol*, char const*, Handle, Handle, Handle, Exceptions::ExceptionMsgToUtf8Mode)+0x1e3
V  [libjvm.so+0x56062f]  Exceptions::_throw_msg(Thread*, char const*, int, Symbol*, char const*, Handle, Handle)+0x7f
V  [libjvm.so+0x67c4c0]  jni_ThrowNew+0x160
C  [libnet.so+0xdd9c]  Java_java_net_PlainSocketImpl_socketAccept+0x16c
j  java.net.PlainSocketImpl.socketAccept(Ljava/net/SocketImpl;)V+0
j  java.net.AbstractPlainSocketImpl.accept(Ljava/net/SocketImpl;)V+7
j  java.net.ServerSocket.implAccept(Ljava/net/Socket;)V+60
J 2961 C1 java.net.ServerSocket.accept()Ljava/net/Socket; (53 bytes) @ 0x00007fffe9612a92 [0x00007fffe9612760+0x332]
j  org.apache.activemq.transport.tcp.TcpTransportServer.doRunWithServerSocket(Ljava/net/ServerSocket;)V+10
j  org.apache.activemq.transport.tcp.TcpTransportServer.run()V+55
j  java.lang.Thread.run()V+11
v  ~StubRoutines::call_stub
V  [libjvm.so+0x662d8b]  JavaCalls::call_helper(JavaValue*, methodHandle*, JavaCallArguments*, Thread*)+0x108b
V  [libjvm.so+0x6641a7]  JavaCalls::call_virtual(JavaValue*, KlassHandle, Symbol*, Symbol*, JavaCallArguments*, Thread*)+0x2f7
V  [libjvm.so+0x664667]  JavaCalls::call_virtual(JavaValue*, Handle, KlassHandle, Symbol*, Symbol*, Thread*)+0x47
V  [libjvm.so+0x6b3661]  thread_entry(JavaThread*, Thread*)+0x91
V  [libjvm.so+0x9fb481]  JavaThread::thread_main_inner()+0xf1
V  [libjvm.so+0x9fb5f0]  JavaThread::run()+0x140
V  [libjvm.so+0x8b7162]  java_start(Thread*)+0x132
C  [libpthread.so.0+0x74a4]  start_thread+0xc4

Java frames: (J=compiled Java code, j=interpreted, Vv=VM code)
v  ~RuntimeStub::resolve_opt_virtual_call
J 3254 C1 java.net.SocketTimeoutException.<init>(Ljava/lang/String;)V (6 bytes) @ 0x00007fffe92a7738 [0x00007fffe92a75e0+0x158]
v  ~StubRoutines::call_stub
j  java.net.PlainSocketImpl.socketAccept(Ljava/net/SocketImpl;)V+0
j  java.net.AbstractPlainSocketImpl.accept(Ljava/net/SocketImpl;)V+7
j  java.net.ServerSocket.implAccept(Ljava/net/Socket;)V+60
J 2961 C1 java.net.ServerSocket.accept()Ljava/net/Socket; (53 bytes) @ 0x00007fffe9612a92 [0x00007fffe9612760+0x332]
j  org.apache.activemq.transport.tcp.TcpTransportServer.doRunWithServerSocket(Ljava/net/ServerSocket;)V+10
j  org.apache.activemq.transport.tcp.TcpTransportServer.run()V+55
j  java.lang.Thread.run()V+11
v  ~StubRoutines::call_stub

---------------  P R O C E S S  ---------------

Java Threads: ( => current thread )
  0x00007fffe403e000 JavaThread "ActiveMQ BrokerService[localhost] Task-3" daemon [_thread_blocked, id=133, stack(0x00007fffc80bc000,0x00007fffc81bd000)]
  0x00007fffe0002000 JavaThread "ConcurrentQueueStoreAndDispatch" daemon [_thread_blocked, id=129, stack(0x00007fffc82be000,0x00007fffc83bf000)]
  0x00007fffc4004000 JavaThread "ActiveMQ Transport: tcp:///172.17.0.1:57574@61616" daemon [_thread_in_native, id=127, stack(0x00007fffc87c1000,0x00007fffc88c2000)]
  0x00007fffd088b800 JavaThread "ActiveMQ InactivityMonitor WriteCheckTimer" daemon [_thread_blocked, id=126, stack(0x00007fffc84c0000,0x00007fffc85c1000)]
  0x00007fffe40f3800 JavaThread "ActiveMQ Transport: tcp:///172.17.0.1:57572@61616" daemon [_thread_in_native, id=125, stack(0x00007fffc83bf000,0x00007fffc84c0000)]
  0x00007fffe403a000 JavaThread "ActiveMQ InactivityMonitor ReadCheckTimer" daemon [_thread_blocked, id=124, stack(0x00007fffc89c3000,0x00007fffc8ac4000)]
  0x00007fffe4175800 JavaThread "ActiveMQ BrokerService[localhost] Task-2" daemon [_thread_blocked, id=115, stack(0x00007fffc88c2000,0x00007fffc89c3000)]
  0x00007fffdc06c800 JavaThread "ActiveMQ BrokerService[localhost] Task-1" daemon [_thread_blocked, id=112, stack(0x00007fffc8bc5000,0x00007fffc8cc6000)]
  0x00007ffff952d800 JavaThread "org.eclipse.jetty.server.session.HashSessionManager@24a1c17fTimer" daemon [_thread_blocked, id=111, stack(0x00007fffc8cc6000,0x00007fffc8dc7000)]
  0x00007ffff9393800 JavaThread "org.eclipse.jetty.server.session.HashSessionManager@f73dcd6Timer" daemon [_thread_blocked, id=110, stack(0x00007fffc8fc7000,0x00007fffc90c8000)]
  0x00007ffff8e05000 JavaThread "qtp1830745997-40" [_thread_blocked, id=109, stack(0x00007fffc92c8000,0x00007fffc93c9000)]
  0x00007ffff8e03000 JavaThread "qtp1830745997-39" [_thread_blocked, id=108, stack(0x00007fffc93c9000,0x00007fffc94ca000)]
  0x00007ffff8e01000 JavaThread "qtp1830745997-38" [_thread_blocked, id=107, stack(0x00007fffc94ca000,0x00007fffc95cb000)]
  0x00007ffff8dff000 JavaThread "qtp1830745997-37" [_thread_blocked, id=106, stack(0x00007fffc95cb000,0x00007fffc96cc000)]
  0x00007ffff8df5000 JavaThread "qtp1830745997-36" [_thread_blocked, id=105, stack(0x00007fffc96cc000,0x00007fffc97cd000)]
  0x00007ffff8df3000 JavaThread "qtp1830745997-35" [_thread_blocked, id=104, stack(0x00007fffc97cd000,0x00007fffc98ce000)]
  0x00007ffff8df0000 JavaThread "qtp1830745997-34-acceptor-0@7850b532-ServerConnector@302fec27{HTTP/1.1}{0.0.0.0:8161}" [_thread_in_native, id=103, stack(0x00007fffc99cf000,0x00007fffc9ad0000)]
  0x00007ffff8df1800 JavaThread "qtp1830745997-33-selector-ServerConnectorManager@6dbc5368/0" [_thread_in_native, id=102, stack(0x00007fffc98ce000,0x00007fffc99cf000)]
  0x00007ffff8bf5800 JavaThread "qtp1856158867-31" [_thread_blocked, id=100, stack(0x00007fffc9cd0000,0x00007fffc9dd1000)]
  0x00007ffff8bf3800 JavaThread "qtp1856158867-30" [_thread_blocked, id=99, stack(0x00007fffc9dd1000,0x00007fffc9ed2000)]
  0x00007ffff8bf1800 JavaThread "qtp1856158867-29" [_thread_blocked, id=98, stack(0x00007fffc9ed2000,0x00007fffc9fd3000)]
  0x00007ffff8bef800 JavaThread "qtp1856158867-28" [_thread_blocked, id=97, stack(0x00007fffc9fd3000,0x00007fffca0d4000)]
  0x00007ffff8bed800 JavaThread "qtp1856158867-27" [_thread_blocked, id=96, stack(0x00007fffca0d4000,0x00007fffca1d5000)]
  0x00007ffff8beb800 JavaThread "qtp1856158867-26" [_thread_blocked, id=95, stack(0x00007fffca1d5000,0x00007fffca2d6000)]
  0x00007ffff8bea000 JavaThread "qtp1856158867-25-acceptor-0@28cdd4ff-ServerConnector@260a3f96{HTTP/1.1}{0.0.0.0:61614}" [_thread_in_native, id=94, stack(0x00007fffca2d6000,0x00007fffca3d7000)]
  0x00007ffff8be9800 JavaThread "qtp1856158867-24-selector-ServerConnectorManager@71b52dc5/0" [_thread_in_native, id=93, stack(0x00007fffca3d7000,0x00007fffca4d8000)]
  0x00007ffff8970000 JavaThread "ActiveMQ Transport Server: mqtt://0.0.0.0:1883?maximumConnections=1000&wireFormat.maxFrameSize=104857600" daemon [_thread_in_native, id=89, stack(0x00007fffca8f5000,0x00007fffca9f6000)]
  0x00007ffff869a800 JavaThread "ActiveMQ Transport Server Thread Handler: mqtt://0.0.0.0:1883?maximumConnections=1000&wireFormat.maxFrameSize=104857600" daemon [_thread_blocked, id=88, stack(0x00007fffca9f6000,0x00007fffcaaf7000)]
=>0x00007ffff859d000 JavaThread "ActiveMQ Transport Server: stomp://0.0.0.0:61613?maximumConnections=1000&wireFormat.maxFrameSize=104857600" daemon [_thread_in_vm, id=87, stack(0x00007fffcaaf7000,0x00007fffcabf8000)]
  0x00007ffff84bd000 JavaThread "ActiveMQ Transport Server Thread Handler: stomp://0.0.0.0:61613?maximumConnections=1000&wireFormat.maxFrameSize=104857600" daemon [_thread_blocked, id=86, stack(0x00007fffcabf8000,0x00007fffcacf9000)]
  0x00007ffff849a800 JavaThread "ActiveMQ Transport Server: amqp://0.0.0.0:5672?maximumConnections=1000&wireFormat.maxFrameSize=104857600" daemon [_thread_in_native, id=85, stack(0x00007fffcacf9000,0x00007fffcadfa000)]
  0x00007ffff8bc8800 JavaThread "ActiveMQ Transport Server Thread Handler: amqp://0.0.0.0:5672?maximumConnections=1000&wireFormat.maxFrameSize=104857600" daemon [_thread_blocked, id=84, stack(0x00007fffcadfa000,0x00007fffcaefb000)]
  0x00007ffff8bca000 JavaThread "ActiveMQ Transport Server: tcp://0.0.0.0:61616?maximumConnections=1000&wireFormat.maxFrameSize=104857600" daemon [_thread_in_native, id=83, stack(0x00007fffcaefb000,0x00007fffcaffc000)]
  0x00007ffff85e9800 JavaThread "ActiveMQ Transport Server Thread Handler: tcp://0.0.0.0:61616?maximumConnections=1000&wireFormat.maxFrameSize=104857600" daemon [_thread_blocked, id=82, stack(0x00007fffcaffc000,0x00007fffcb0fd000)]
  0x00007ffff8bc5000 JavaThread "ActiveMQ Data File Writer" daemon [_thread_blocked, id=77, stack(0x00007fffcb2fd000,0x00007fffcb3fe000)]
  0x00007ffff8bbf800 JavaThread "ActiveMQ Journal Checkpoint Worker" daemon [_thread_blocked, id=76, stack(0x00007fffcb3fe000,0x00007fffcb4ff000)]
  0x00007ffff8ba1800 JavaThread "ActiveMQ Journal Scheduled executor" daemon [_thread_blocked, id=73, stack(0x00007fffcb4ff000,0x00007fffcb600000)]
  0x00007ffff8b89000 JavaThread "ActiveMQ Broker[localhost] Scheduler" daemon [_thread_blocked, id=72, stack(0x00007fffd8075000,0x00007fffd8176000)]
  0x00007ffff81ff800 JavaThread "RMI TCP Accept-0" daemon [_thread_in_native, id=42, stack(0x00007fffd85c7000,0x00007fffd86c8000)]
  0x00007ffff80dd000 JavaThread "Service Thread" daemon [_thread_blocked, id=36, stack(0x00007fffdbcfd000,0x00007fffdbdfe000)]
  0x00007ffff80da000 JavaThread "C1 CompilerThread1" daemon [_thread_blocked, id=35, stack(0x00007fffdbdff000,0x00007fffdbeff000)]
  0x00007ffff80d7800 JavaThread "C2 CompilerThread0" daemon [_thread_blocked, id=34, stack(0x00007fffdbf00000,0x00007fffdc000000)]
  0x00007ffff80d6000 JavaThread "Signal Dispatcher" daemon [_thread_blocked, id=33, stack(0x00007fffe8013000,0x00007fffe8114000)]
  0x00007ffff80a2000 JavaThread "Finalizer" daemon [_thread_blocked, id=32, stack(0x00007fffe82b8000,0x00007fffe83b9000)]
  0x00007ffff809f800 JavaThread "Reference Handler" daemon [_thread_blocked, id=30, stack(0x00007fffe83b9000,0x00007fffe84ba000)]
  0x00007ffff800b000 JavaThread "main" [_thread_blocked, id=20, stack(0x00007ffffd892000,0x00007ffffd992000)]

Other Threads:
  0x00007ffff8096000 VMThread [stack: 0x00007fffe84bb000,0x00007fffe85bb000] [id=29]
  0x00007ffff8215000 WatcherThread [stack: 0x00007fffd84c7000,0x00007fffd85c7000] [id=44]

...

VM Arguments:
jvm_args: -Xms64M -Xmx1G -Djava.util.logging.config.file=logging.properties -Djava.security.auth.login.config=/opt/activemq/conf/login.config -Dcom.sun.management.jmxremote -Djava.awt.headless=true -Djava.io.tmpdir=/opt/activemq/tmp -Dactivemq.classpath=/opt/activemq/conf:/opt/activemq/../lib/: -Dactivemq.home=/opt/activemq -Dactivemq.base=/opt/activemq -Dactivemq.conf=/opt/activemq/conf -Dactivemq.data=/opt/activemq/data 
java_command: /opt/activemq/bin/activemq.jar start
java_class_path (initial): /opt/activemq/bin/activemq.jar
Launcher Type: SUN_STANDARD

Environment Variables:
JAVA_HOME=/usr/local/openjdk-8
PATH=/usr/local/openjdk-8/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin

Signal Handlers:
SIGSEGV: [libjvm.so+0xa485f0], sa_mask[0]=11111111111111111111111111111110, sa_flags=SA_RESTART|SA_SIGINFO
SIGBUS: [libjvm.so+0xa485f0], sa_mask[0]=11111111111111111111111111111110, sa_flags=SA_RESTART|SA_SIGINFO
SIGFPE: [libjvm.so+0x8b05f0], sa_mask[0]=11111111111111111111111111111110, sa_flags=SA_RESTART|SA_SIGINFO
SIGPIPE: [libjvm.so+0x8b05f0], sa_mask[0]=11111111111111111111111111111110, sa_flags=SA_RESTART|SA_SIGINFO
SIGXFSZ: [libjvm.so+0x8b05f0], sa_mask[0]=11111111111111111111111111111110, sa_flags=SA_RESTART|SA_SIGINFO
SIGILL: [libjvm.so+0x8b05f0], sa_mask[0]=11111111111111111111111111111110, sa_flags=SA_RESTART|SA_SIGINFO
SIGUSR1: SIG_DFL, sa_mask[0]=00000000000000000000000000000000, sa_flags=none
SIGUSR2: [libjvm.so+0x8b1da0], sa_mask[0]=00000000000000000000000000000000, sa_flags=SA_RESTART|SA_SIGINFO
SIGHUP: [libjvm.so+0x8b37d0], sa_mask[0]=11111111111111111111111111111110, sa_flags=SA_RESTART|SA_SIGINFO
SIGINT: [libjvm.so+0x8b37d0], sa_mask[0]=11111111111111111111111111111110, sa_flags=SA_RESTART|SA_SIGINFO
SIGTERM: [libjvm.so+0x8b37d0], sa_mask[0]=11111111111111111111111111111110, sa_flags=SA_RESTART|SA_SIGINFO
SIGQUIT: [libjvm.so+0x8b37d0], sa_mask[0]=11111111111111111111111111111110, sa_flags=SA_RESTART|SA_SIGINFO


---------------  S Y S T E M  ---------------

OS:PRETTY_NAME="Debian GNU/Linux 9 (stretch)"
NAME="Debian GNU/Linux"
VERSION_ID="9"
VERSION="9 (stretch)"
ID=debian
HOME_URL="https://www.debian.org/"
SUPPORT_URL="https://www.debian.org/support"
BUG_REPORT_URL="https://bugs.debian.org/"

uname:Linux 6.6.26-linuxkit #1 SMP Sat Apr 27 04:13:19 UTC 2024 x86_64
libc:glibc 2.24 NPTL 2.24 
rlimit: STACK 8192k, CORE 0k, NPROC infinity, NOFILE 1048576, AS infinity
load average:1.99 0.89 0.35

/proc/meminfo:
MemTotal:        9150200 kB
MemFree:         5362880 kB
MemAvailable:    7506248 kB
Buffers:           53288 kB
Cached:          2171248 kB
SwapCached:            0 kB
Active:          1526688 kB
Inactive:        2003132 kB
Active(anon):    1322804 kB
Inactive(anon):        0 kB
Active(file):     203884 kB
Inactive(file):  2003132 kB
Unevictable:           0 kB
Mlocked:               0 kB
SwapTotal:       1048572 kB
SwapFree:        1048572 kB
Zswap:                 0 kB
Zswapped:              0 kB
Dirty:              4884 kB
Writeback:             0 kB
AnonPages:       1297796 kB
Mapped:           222164 kB
Shmem:             17460 kB
KReclaimable:     143700 kB
Slab:             189016 kB
SReclaimable:     143700 kB
SUnreclaim:        45316 kB
KernelStack:        6768 kB
PageTables:         9052 kB
SecPageTables:         0 kB
NFS_Unstable:          0 kB
Bounce:                0 kB
WritebackTmp:          0 kB
CommitLimit:     5623672 kB
Committed_AS:    4949136 kB
VmallocTotal:   133141626880 kB
VmallocUsed:        2480 kB
VmallocChunk:          0 kB
Percpu:              324 kB
AnonHugePages:    706560 kB
ShmemHugePages:        0 kB
ShmemPmdMapped:        0 kB
FileHugePages:         0 kB
FilePmdMapped:         0 kB
HugePages_Total:       0
HugePages_Free:        0
HugePages_Rsvd:        0
HugePages_Surp:        0
Hugepagesize:       2048 kB
Hugetlb:               0 kB


CPU:total 1 (initial active 1) (1 cores per cpu, 1 threads per core) family 6 model 44 stepping 0, cmov, cx8, fxsr, mmx, sse, sse2, sse3, ssse3, sse4.1, sse4.2, popcnt, aes, clmul, tsc, tscinvbit, tscinv

/proc/cpuinfo:
processor: 0
BogoMIPS: 48.00
Features: fp asimd evtstrm aes pmull sha1 sha2 crc32 atomics fphp asimdhp cpuid asimdrdm jscvt fcma lrcpc dcpop sha3 asimddp sha512 asimdfhm dit uscat ilrcpc flagm ssbs sb paca pacg dcpodp flagm2 frint
CPU implementer: 0x61
CPU architecture: 8
CPU variant: 0x0
CPU part: 0x000
CPU revision: 0



Memory: 4k page, physical 9150200k(5362880k free), swap 1048572k(1048572k free)

vm_info: OpenJDK 64-Bit Server VM (25.212-b04) for linux-amd64 JRE (1.8.0_212-b04), built on May 21 2019 14:20:37 by "openjdk" with gcc 4.4.7 20120313 (Red Hat 4.4.7-23)

time: Wed Jun  5 21:54:01 2024
timezone: UTC
elapsed time: 135 seconds (0d 0h 2m 15s)

root@ad5e5b71d598 /opt/apache-activemq-5.15.9 [quizzical_wilbur]
docker > 
```

**Comment by thaJeztah** [comment]
Possibly duplicate of https://github.com/docker/for-mac/issues/7006 and if it is; may be an Apple bug that we reported to them; https://github.com/docker/for-mac/issues/7006#issuecomment-2122869966

One workaround is to disable Rosetta; https://github.com/docker/for-mac/issues/7006#issuecomment-2124954348


cc @doringeman

**Comment by doringeman** [comment]
Yes, @thaJeztah is right.
Thanks for your report!

**Comment by rafaelekol** [comment]
Awful bug
